### PR TITLE
Reorganizing the setup-your-client-environment page

### PR DESCRIPTION
**Pull request by yevgenykhazan**
Please don't merge, just review for now

### CODE REVIEW

File: docs/install/remote.md [hunk 1]
```diff
@@ -50,67 +50,45 @@ Before you begin, ensure that the following prerequisites are met:
...
-
-```python
-# Use local service
-mlrun.set_environment("http://localhost:8080", artifact_path="./")
-# Use remote service
-mlrun.set_environment("<remote-service-url>", access_key="xyz", username="joe")
-```
-
-3. Run `mlrun config set` command line to set configuration parameters in mlrun default or specified .env file. By default, it stores all of the configuration into the default environment file, and your own environment file does not need editing.
+Run `mlrun config set` command line to set configuration parameters in mlrun default or specified `.env` file. By default, it stores all of the configuration into the default environment file, and your own environment file does not need editing.
```
Comment (jillnogold) [inline docs/install/remote.md:72]: Run the `mlrun config set` command line to set configuration parameters in the mlrun default or the specified `.env` file. By default, it stores all of the configuration into the default environment file, and your own environment file does not need editing.

File: docs/install/remote.md [hunk 2]
```diff
@@ -144,19 +122,50 @@ project.set_secrets(file_path=env_file)
...
+    any page, and select **Access Keys** from the menu. In the **Access Keys** window, either copy an existing access key or create a new 
+    key and copy it. Alternatively, you can get the access key by checking the value of the `V3IO_ACCESS_KEY` environment variable in a web-shell or Jupyter Notebook service.
+
+
+**You can also set the environment using MLRun CLI, for example:**
 
 1. Create an env file similar to the example, with lines in the form KEY=VALUE, and comment lines starting with "#".
 2. Use `--env-file <env file path>` in mlrun run/build/deploy/project CLI commands to load the config and credential env vars from file.
-2. Set the `MLRUN_ENV_FILE=<env file path>` env var to point to a default env file (which will be loaded on import).
+3. Set the `MLRUN_ENV_FILE=<env file path>` env var to point to a default env file (which will be loaded on import).
```
Comment (jillnogold) [inline docs/install/remote.md:160]: 3. Set the `MLRUN_ENV_FILE=<env file path>` env var to point to a default env file (which load on import).
   If the `MLRUN_DBPATH` points to a remote iguazio cluster and the `V3IO_API` and/or `V3IO_FRAMESD` vars are not set, they are inferred from the DBPATH.

File: docs/install/remote.md [hunk 3]
```diff
@@ -50,67 +50,45 @@ Before you begin, ensure that the following prerequisites are met:
...
+3. Alternatively, if you already installed a previous version of MLRun, upgrade it by running:
 
     ```sh
     pip install -U mlrun==<version>
     ```
 
-2. Ensure that you have remote access to your MLRun service (i.e., to the service URL on the remote Kubernetes cluster).
+4. Ensure that you have remote access to your MLRun service (i.e., to the service URL on the remote Kubernetes cluster).
 
 ## Configure remote environment
```
Comment (jillnogold) [inline docs/install/remote.md:61]: This list is already in "In this section," and adding this heading pushes all the heading levels down. I suggest either removing it, or renaming it overview and changing the heading level (up one level) of the sections that start with **Using**.
Also there's mismatch in terminology. The heading is client environment, and the list is remote environment. Change the title of the topic?
image

File: docs/install/remote.md [hunk 4]
```diff
@@ -50,67 +50,45 @@ Before you begin, ensure that the following prerequisites are met:
...
+You have few options to configure your remote environment:
+1. Using the default `.env` file (**Recommended**)
+  - The default `.env` file is located by default at `~/.mlrun.env` for Linux and `%USERPROFILE%/.mlrun.env` for Windows
```
Comment (jillnogold) [inline docs/install/remote.md:64]: move these lines to before line 72

File: docs/install/remote.md [hunk 5]
```diff
@@ -50,67 +50,45 @@ Before you begin, ensure that the following prerequisites are met:
...
+Run `mlrun config set` command line to set configuration parameters in mlrun default or specified `.env` file. By default, it stores all of the configuration into the default environment file, and your own environment file does not need editing.
 
 The `set` command can work with the following parameters:
-    `--env-file` or `-f` to set the url path to the mlrun .env file (defaults to '~/.mlrun.env' for Linux and to %USERPROFILE%/.mlrun.env for Windows)
-    `--api` or `-a` to set the url (local or remote) for MLRun API
-    `--artifact-path` or `-p` to set the artifact path (optional)
-    `--username` or `-u` to set the username (for remote access)
-    `--access-key` or `-k` to set the access key (for remote access)
-    `--env-vars` or `-e` to set the environment variables (optional)
+    - `--env-file` or `-f` to set the url path to the mlrun `.env file`
```
Comment (jillnogold) [inline docs/install/remote.md:75]: - `--env-file` or `-f` to set the url path to the mlrun `.env file` 
- `--api` or `-a` to set the url (local or remote) for MLRun API
- `--artifact-path` or `-p` to set the artifact path (optional)
- `--username` or `-u` to set the username (for remote access)
- `--access-key` or `-k` to set the access key (for remote access)
- `--env-vars` or `-e` to set the environment variables (optional)

File: docs/install/remote.md (hunk 1 above)
Comment (yevgenykhazan) [reply docs/install/remote.md:72]: fixed

File: docs/install/remote.md (hunk 2 above)
Comment (yevgenykhazan) [reply docs/install/remote.md:160]: fixed

File: docs/install/remote.md (hunk 3 above)
Comment (yevgenykhazan) [reply docs/install/remote.md:61]: fixed

File: docs/install/remote.md (hunk 4 above)
Comment (yevgenykhazan) [reply docs/install/remote.md:64]: fixed

File: docs/install/remote.md (hunk 5 above)
Comment (yevgenykhazan) [reply docs/install/remote.md:75]: fixed

**Comment by yevgenykhazan** [comment]
Opened a new PR https://github.com/mlrun/mlrun/pull/3011 with Jill's comments implemented

**Pull request closed without merging**